src/app/contact/page.jsx

```diff
@@ -0,0 +1,12 @@
+export default function ContactPage() {
+    return (
+    <div style={{
+        display:'flex',
+        justifyContent:'center',
```

Comment:
We are currently using css modules for styling, make sure to stick to a single coding pattern.